src/components/HomePageContent/HowItWorksSection/index.tsx

```diff
@@ -21,7 +21,7 @@ export const HowItWorksSection: FunctionComponent = () => {
   const documentTypeFilterStyle = (item: DocumentType): string => {
     let returnStyle;
 
-    if (item === selectedDocumentTypeLabel) returnStyle = "font-bold underline";
+    if (item === selectedDocumentTypeLabel) returnStyle = "font-gilroy-bold underline";
```

Comment:
same?

Reply:
dont think this can be able to do that, cos this function look more like a nested if statement as of now. so if put into switch statement it will  be more messy.

Reply:
on another hand, why is verifiable hidden in mobile?

Reply:
i asked jerry about the verifiable hidden on mobile, he said its due to the design and dont want use to have too many buttons on mobile.